include/vctr/Containers/VctrBase.h

```diff
@@ -251,7 +253,42 @@ class VctrBase : private Config,
     }
 
     /** Fills the container with the given value. */
-    constexpr void fill (const value_type& value) {  std::fill (begin(), end(), value); }
+    constexpr void fill (const value_type& value) { std::fill (begin(), end(), value); }
+
+    //==============================================================================
+    // Generators
+    //==============================================================================
+    /** Fills the vector with evenly spaced numbers between start and stop.
+
+        The length of the vector will not be changed and must be > 0.
+        If start and stop are identical, the value will be filled with that value.
+
+        @param start      First value of the interval; always included.
+        @param stop       Last value of the interval; only included if includeEnd is true.
+        @param includeEnd If this is true, the numbers lie on the closed interval
+                          [start, stop]; otherwise the interval is half-open [start, stop).
+     */
+    void fillLinspace (ElementType start, ElementType stop, bool includeEnd = true)
+    requires is::number<ElementType>
+    {
+        const auto num = size();
+
+        VCTR_ASSERT (num > 0);
+        VCTR_ASSERT (! includeEnd || num != 1); // a num-one range cannot include the end
```

Comment:
Should we assert that `start < stop`?

Reply:
No. If you have a look at the tests, you'll see that we can go in both directions.

Reply:
Ah right, wrote this before going through the test in detail.

Reply:
I wonder if we should explicitly mention this feature in the documentation?

Reply:
Sure, why not :)

Reply:
Than also means it's not correct to define the interval as [start, stop) which would only be true if start < stop